docker-compose.yaml

```diff
@@ -97,51 +97,15 @@ services:
       options:
         max-size: "2048m"
 
-  raw-backfiller:
-    container_name: raw-backfiller
-    restart: always
-    entrypoint: sh -c "if [ -z '$$MALLOC_CONF' ]; then exec ./raw_backfiller; else exec ./profiling_raw_backfiller; fi"
-    env_file:
-      - .env
-    network_mode: host
-    volumes:
-      - ${INGESTER_ROCKS_DB_PATH}:${INGESTER_ROCKS_DB_PATH_CONTAINER}:rw
-      - ${INGESTER_PROFILING_FILE_PATH}:${INGESTER_PROFILING_FILE_PATH_CONTAINER}:rw
-      - ./creds.json:/usr/src/app/creds.json
-      - ./heaps:/usr/src/app/heaps:rw
-    stop_grace_period: 5m
-    build:
-      context: .
-      dockerfile: ingester.Dockerfile
-    logging:
-      options:
-        max-size: "2048m"
-
   slot-persister:
     container_name: slot-persister
     restart: always
-    entrypoint: |
-      sh -c "
-      ARGS=\"--target-db-path $target_db_path\"
-      ARGS=\"$$ARGS --rpc-host $rpc_host\"
-      [ -n \"$start_slot\" ] && ARGS=\"$$ARGS --start-slot $start_slot\"
-      [ -n \"$big_table_credentials\" ] && ARGS=\"$$ARGS --big-table-credentials $big_table_credentials\"
-      [ -n \"$big_table_timeout\" ] && ARGS=\"$$ARGS --big-table-timeout $big_table_timeout\"
-      [ -n \"$metrics_port\" ] && ARGS=\"$$ARGS --metrics-port $metrics_port\"
-      [ -n \"$chunk_size\" ] && ARGS=\"$$ARGS --chunk-size $chunk_size\"
-      [ -n \"$max_concurrency\" ] && ARGS=\"$$ARGS --max-concurrency $max_concurrency\"
-
-      if [ -z \"$MALLOC_CONF\" ]; then
-        exec ./slot_persister $$ARGS
-      else
-        exec ./profiling_slot_persister $$ARGS
-      fi"
+    entrypoint: "./slot_persister"
     env_file:
       - .env
     network_mode: host
     volumes:
-      - ${target_db_path}:${target_db_path}:rw
-      - ${INGESTER_PROFILING_FILE_PATH}:${INGESTER_PROFILING_FILE_PATH_CONTAINER}:rw
+      - ${TARGET_DB_PATH}:${TARGET_DB_PATH}:rw
```

Comment:
value missing from .env.example. 
Also some conflicts with the #310 are expected